zerver/openapi/python_examples.py

```diff
@@ -223,23 +223,15 @@ def get_single_user(client):
     # {code_example|start}
     # Fetch details on a user given a user ID
     user_id = 8
-    url = 'users/' + str(user_id)
-    result = client.call_endpoint(
-        url=url,
-        method='GET'
-    )
+    result = client.get_user_by_id(user_id)
     # {code_example|end}
     validate_against_openapi_schema(result, '/users/{user_id}', 'get', '200')
 
     # {code_example|start}
     # If you'd like data on custom profile fields, you can request them as follows:
-    result = client.call_endpoint(
-        url=url,
-        method='GET',
-        request={'include_custom_profile_fields': True}
-    )
+    request = {'include_custom_profile_fields': True}
+    result = client.get_user_by_id(user_id, request)
```

Comment:
I don't think we need a separate `request` variable; just pass the dictionary as the second argument directly.  I think it's more readable that way, in part because `request` probably isn't what a user would call that field; likely something more like "parameters".  But the best answer is to just not name the argument at all ourselves.